### PR TITLE
feat: make template components visible in the client

### PR DESCRIPTION
**Pull request by mohamed-cds**
Can now view your organisation, it's templates, and then configure scans for each template.

### CODE REVIEW

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

File: api/front_end/templates/user.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+{% block includes %}
+<script src="{{ url_for('static', path='js/user.js') }}"></script>
+{% endblock %}
+{% block body %}
+  
+    <div class="container flex justify-center items-center">
+      <div>
+        <h1>{{ request.session.user.name }}</h1>
+        <p><b>{{ email_locale }}:</b> {{ request.session.user.email_address }}</p>
```
Comment (patheard) [inline api/front_end/templates/user.html:10]: Minor tweak: should use `<strong>` instead of `<b>`

File: api/front_end/templates/user.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <div id='child'>
+        <b>{{ api_key_locale }}:</b>
```
Comment (patheard) [inline api/front_end/templates/user.html:12]: Same comment re: `<strong>`

**Review by patheard — CHANGES_REQUESTED** [review]
Two minor changes in the HTML markup, but otherwise, good stuff!

**Comment by patheard** [comment]
One other quest I have that's probably outside the scope of this PR is can we standardize on FontAwesome or Google Material icons?

**Review by patheard — APPROVED** [review]

**Comment by mohamed-cds** [comment]
> One other question I have that's probably outside the scope of this PR is can we standardize on FontAwesome or Google Material icons?

Removed FontAwesome since i was only using it for the google icon which was barely visible!

**Pull request merged**